### PR TITLE
Fix events overlay

### LINKED ISSUE #1920: Send events overlay background broken
## Bug description

This image doesn't load:

https://posthog.s3.eu-west-2.amazonaws.com/preview-action-trends.png?1602965696886

image

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

*Please describe.*  
*If this affects the front-end, include screenshots.*  

- Fix #1920 
- Fix responsiveness on text

Screenshot 2020-10-18 at 11 17 12

## Checklist

- [ ] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [ ] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

**Comment by mariusandra** [comment]
There's still a background image that doesn't load. See the dotted line/box on the top/right of the white box.

**Comment by yakkomajuri** [comment]
Will look into it

**Comment by yakkomajuri** [comment]
Any idea what that image was? Seems to not be found on S3

**Comment by yakkomajuri** [comment]
We can also ditch the image for a more prominent overlay? 

Screenshot 2020-10-18 at 11 45 27

**Comment by mariusandra** [comment]
I think the image was some gradient, though you'll need to ask @timgl to be sure.
Totally fine with me to remove it altogether and leave it live your last screenshot. Just make sure to root out the `image` prop from its source, unless it's used by something else...

**Comment by yakkomajuri** [comment]
Ahh I remember now! We used to show a preview of like charts/events etc in the background. Will see if I can figure out what happened to those pics otherwise will submit a fix that just gets rid of them altogether (perhaps temporarily?)

**Comment by yakkomajuri** [comment]
Seems like the bucket was deleted

**Comment by yakkomajuri** [comment]
So the bucket doesn't exist and I couldn't track it down since we don't have any versioning set up on S3 or anything.

Hence, I actually got rid of both the image and the overlay, because there's no need for an overlay on a blank page. 

Left it as a comment though for it to be quickly re-enabled with the right color.

Screenshot 2020-10-18 at 12 21 40

Also checked the repo for any other assets loading from that deleted bucket and we don't seem to have any, which is good.

**Comment by yakkomajuri** [comment]
Tagging @EDsCODE here because of https://github.com/PostHog/posthog/pull/1077/ in case you know what those pictures were so we can add them to S3 again.

**Comment by EDsCODE** [comment]
Huh, I didn't have any of the original images locally either. When adding the default picture of sessions view I just reused one of the links because it worked. Not sure where the bucket went!

File: frontend/src/style/send-events-overlay.scss [hunk 1]
```diff
@@ -27,9 +26,8 @@
     }
     .overlay {
         width: 100%;
-        height: 100%;
         padding: 5rem;
-        background: linear-gradient(125deg, rgb(255, 255, 255) 25%, rgba(255, 255, 255, 0) 100%);
+        /* background: linear-gradient(125deg, rgb(243, 243, 243) 25%, rgba(255, 255, 255, 0) 100%); */
```
Comment (mariusandra) [inline frontend/src/style/send-events-overlay.scss:30]: Just a [small nit](https://kentcdodds.com/blog/please-dont-commit-commented-out-code) here :D

**Review by mariusandra — APPROVED** [review]
Great, just one small inline nit :).

**Pull request merged**